packages/lib/date-ranges.test.ts

```diff
@@ -159,13 +159,13 @@ describe("processWorkingHours", () => {
     // in America/New_York DST ends on first Sunday of November
     const timeZone = "America/New_York";
 
-    let firstSundayOfNovember = dayjs().startOf("day").month(10).date(1);
+    let firstSundayOfNovember = dayjs.utc().startOf("day").month(10).date(1);
     while (firstSundayOfNovember.day() !== 0) {
       firstSundayOfNovember = firstSundayOfNovember.add(1, "day");
     }
 
-    const dateFrom = dayjs().month(10).date(1).startOf("day");
-    const dateTo = dayjs().month(10).endOf("month");
+    const dateFrom = dayjs.utc().month(10).date(1).startOf("day");
+    const dateTo = dayjs.utc().month(10).endOf("month");
```

Comment:
using dayjs.utc to avoid change of date ( incremented to next day )
while using dayjs.endOf in certain timezones